### ISSUE TITLE
getVulkanVersion ValueError

### ISSUE THREAD

**Issue by andreymal** [arunsivaramanneo/GPU-Viewer#21]
```
Traceback (most recent call last):
  File "/usr/share/gpu-viewer/Files/VulkanViewer.py", line 681, in radcall
    Instance()
  File "/usr/share/gpu-viewer/Files/VulkanViewer.py", line 538, in Instance
    [line.strip('\n'), Vversion[i].strip('\n'), getVulkanVersion(LVersion[i]).strip('\n'),
  File "/usr/share/gpu-viewer/Files/Common.py", line 151, in getVulkanVersion
    majorVersion = int(value) >> 22
ValueError: invalid literal for int() with base 10: ' 1.1.73\n'
```

vulkaninfo.txt content: https://pastebin.com/E62JJp6k

**Comment by arunsivaramanneo** [comment]
Thanks for repoting this issue .. I have fixed it ... Will release an updated version soon ... Currently the Format section is completely broken... so if you are using the latest version please do not that Formats section will not be displayed

**Comment by arunsivaramanneo** [comment]
This is fixed with release v1.16. Closing the defect
